### PR TITLE
add new files per mozorg PR175

### PR DESCRIPTION
**Pull request by peiying2**
Add three files/deadlines 
- sendto-tabs
- plugincheck-update
- batm (need to confirm if this the file name)

Obsolete
- sendto
- obsolete-redesign

### CODE REVIEW

File: app/config/sources.inc.php [hunk 1]
```diff
@@ -316,6 +321,13 @@
         'supported_locales' => $mozillaorg,
     ],
     'firefox/sendto.lang' => [
+        'flags'             => [
+          'obsolete'        => ['all'],
```
Comment (flodolo) [inline app/config/sources.inc.php]: Please fix the indentation here (4 spaces, not 1)

File: app/config/sources.inc.php [hunk 2]
```diff
@@ -588,6 +600,13 @@
         'supported_locales' => $newsletter_locales,
     ],
     'mozorg/plugincheck-redesign.lang' => [
+        'flags'             => [
+          'obsolete'        => ['all'],
```
Comment (flodolo) [inline app/config/sources.inc.php]: Same here

File: app/config/sources.inc.php [hunk 3]
```diff
@@ -220,6 +220,11 @@
         'priority'          => 1,
         'supported_locales' => $mozillaorg,
     ],
+    'firefox/features/sendto-tabs.lang' => [
+        'deadline'          => '2017-08-15',
+        'priority'          => 1,
+        'supported_locales' => $firefox_locales,
```
Comment (flodolo) [inline app/config/sources.inc.php]: Other pages are exposed to all mozilla.org, let's keep it consistent

File: app/config/sources.inc.php [hunk 4]
```diff
@@ -316,8 +321,10 @@
         'supported_locales' => $mozillaorg,
     ],
     'firefox/sendto.lang' => [
-        'priority'          => 1,
+        'flags'             => [
+          'obsolete'        => ['all'],
```
Comment (flodolo) [inline app/config/sources.inc.php]: The indentation is wrong, but more important, this change is **really** wrong. We're adding a page that describe the "Send to" feature, this file is the widget to send SMS on mozilla.org…

File: app/config/sources.inc.php [hunk 5]
```diff
@@ -588,6 +595,13 @@
         'supported_locales' => $newsletter_locales,
     ],
     'mozorg/plugincheck-redesign.lang' => [
+        'flags'             => [
+          'obsolete'        => ['all'],
+        'supported_locales' => $mozillaorg,
```
Comment (flodolo) [inline app/config/sources.inc.php]: Check this array and try to figure out why it's wrong, where parenthesis open and close.

**Comment by flodolo** [comment]
One more thing: since it's a template using firefox/shared, you need to add it to the script used to activate those pages.

File: app/config/sources.inc.php (hunk 4 above)
Comment (peiying2) [inline app/config/sources.inc.php]: change array to firefox locales.

File: app/config/sources.inc.php [hunk 6]
```diff
@@ -316,8 +321,10 @@
         'supported_locales' => $mozillaorg,
     ],
     'firefox/sendto.lang' => [
-        'priority'          => 1,
+        'flags'             => [
+          'obsolete'    => ['all'],
```
Comment (flodolo) [inline app/config/sources.inc.php]: Once more: this is NOT an obsolete file.

Also, these arrays should look like:

```
'mozorg/plugincheck-redesign.lang' => [
    'flags'             => [
        'obsolete' => ['all'],
    'supported_locales' => $mozillaorg,
    ],
],
```

File: app/config/sources.inc.php [hunk 7]
```diff
@@ -316,8 +321,10 @@
         'supported_locales' => $mozillaorg,
     ],
     'firefox/sendto.lang' => [
+        'flags'             => [
         'priority'          => 1,
         'supported_locales' => $firefox_locales,
+        ],
```
Comment (flodolo) [inline app/config/sources.inc.php]: This is wrong, check other files and try to understand what the structure means

File: app/config/sources.inc.php [hunk 8]
```diff
@@ -588,6 +595,13 @@
         'supported_locales' => $newsletter_locales,
     ],
     'mozorg/plugincheck-redesign.lang' => [
+        'flags'             => [
+          'obsolete' => ['all'],
+        'supported_locales' => $mozillaorg,
```
Comment (flodolo) [inline app/config/sources.inc.php]: Wrong structure

**Pull request closed without merging**